front-end-admin/src/api/ui-management.ts

```
@@ -0,0 +1,176 @@
+
+export const useUpdateRewardSettings = (): UseMutationResult<any> => {
+  const queryClient = useQueryClient()
+
+  return useMutation({
+    mutationKey: 'update_reward_settings',
+    mutationFn: async (data: any) => {
+      try {
+        const response = await authFetch.put(`/rewards-service-settings` , data)
+        return response.data
+      } catch (error) {
+        throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)